main.go

```
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"strconv"
+	"strings"
+)
+
+func TCPServer() {

```

I would recommend to return an error from this function instead of log.Fatal